api/agent/lb_agent.go

```diff
@@ -129,22 +92,23 @@ const (
 	// sleep time when scaling from 0 to 1 runners
 	noCapacityWaitInterval = 1 * time.Second
 	// amount of time to wait to place a request on a runner
-	placementTimeout = 15 * time.Second
+	placementTimeout          = 15 * time.Second
+	runnerPoolShutdownTimeout = 5 * time.Second
 )
 
 type lbAgent struct {
 	delegatedAgent Agent
-	np             NodePool
+	rp             models.RunnerPool
```

Comment:
Sorry I should have said this on the previous PR - is `models.` the right package for the RunnerPool? - models would seem to be the place for things related to persistence and exchange of data, RunnerPool would seem to either sit on its own or somewhere else.

Reply:
I moved all interfaces to api/runnerpool.